shared/js/media/remote_controls.js

```diff
@@ -368,14 +338,14 @@ MediaRemoteControls.prototype.notifyAppInfo = function(info) {
  * @param {Object} metadata
  */
 MediaRemoteControls.prototype.notifyMetadataChanged = function(metadata) {
+  var onsuccess = function() {
+    // To do nothing here, expect onsuccess normally.
+  };
+  var onerror = function() {
+    console.log('Sending Metadata error');
+  };
```

Comment:
Let's keep the function pattern in this module like:

```JavaScript
function onerror() {
  console.log('Sending Metadata error');
};
```

instead of

```JavaScript
var onerror = function() {
  console.log('Sending Metadata error');
};

```